### PR TITLE
Lesson 10

### PR DESCRIPTION
**Pull request by DmitriiTeliuk**
Created constants and pages folder
Refactored tests
Added logging
Added waiters and decorator for Sign in click method

### CODE REVIEW

File: Constance/base.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class BaseConst:
```
Comment (QALightLecorPython) [inline Constance/base.py]: Typo: Constance is a name ;) 
constants is a directory. Also we're naming python packages using lowercase

File: pages/base_page.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        user_field.clear()
+        user_field.send_keys(some_value)
+
+    def wait_until_find_element(self, by, value):
+        """Wait unlit element is found"""
+        return self.wait.until(EC.presence_of_element_located(locator=(by, value)))
+
+    def wait_until_find_visible_and_clickable(self, by, value):
+        element = self.wait_until_find_element(by, value)
+        return self.wait.until(EC.element_to_be_clickable((by, value)))
```
Comment (QALightLecorPython) [inline pages/base_page.py]: No new line at the end of file

**Review by QALightLecorPython — APPROVED** [review]
Looks good! 
You can merge now, but please solve described issues in the next PR

**Comment by DmitriiTeliuk** [comment]
fixed issue with folder name 'Constance' -> constance
added new line in base_page

**Pull request merged**